packages/terra-worklist-data-grid/src/subcomponents/ColumnHeader.jsx

```diff
@@ -36,6 +38,11 @@ const propTypes = {
   onResizeMouseDown: PropTypes.func,
 };
 
+const defaultProps = {
+  pinnedColumns: [],
+  overflowColumns: [],
```

Comment:
These don't appear to be props of the ColumnHeader component.

Reply:
You're right, most of the changes in this file were left over from our original changes. Reverted here: [1a46886](https://github.com/cerner/terra-framework/pull/1710/commits/1a4688675452581e97d235eee07f053457a24860)